lib/sync-records.ts

```diff
@@ -67,12 +67,34 @@ export const syncWorkbookRecords = async ({
     });
   }
 
+  type RetryData = [
+    {
+      organizationId: string;
+      employeeId: string;
+      employeeTypeId: string;
+      firstName: string;
+      lastName: string;
+      hireDate: Date;
+      endEmploymentDate: Date | null;
+      positionTitle: string;
+      defaultWeeklyHours: number;
+      scheduledWeeklyHours: number;
+      flatfileRecordId?: string | undefined;
+      jobId: string;
+    },
+    {
+      recordManagerId: string;
+    }
+  ];
+
   if (employeeRecords) {
     // TODO: Refactor employee logic to its lib file
     const validsManagersFirst = employeeRecords?.sort((a, b) => {
       return a.values.managerId.value ? 1 : -1;
     });
 
+    let employeesWithoutMangersInDB: Array<RetryData> = [];
```

Comment:
Nitpick: I am used to see `RetryData[]` more

Reply:
Edited